nuitka/tools/data_composer/DataComposer.py

```diff
@@ -135,9 +135,9 @@ def _writeConstantValue(output, constant_value):
             _writeConstantValue(output, element)
     elif constant_type is long:
         if isPythonValidCLongValue(constant_value):
-            output.write(b"l" + struct.pack("l", constant_value))
+            output.write(b"l" + struct.pack("=l", constant_value))
```

Comment:
I am confused why the representation would have to change without a change on the decoding side. Are we saying that network byte order vs. native order is something the decoder doesn't do properly?

Reply:
I now figured out, that you didn't run Nuitka on WASI, which is fine by me, but normally would be the recommended approach.